platform/android/java_godot_io_wrapper.h

```diff
@@ -45,6 +45,8 @@ class GodotIOJavaWrapper {
 	jobject godot_io_instance;
 	jclass cls;
 
+	jmethodID _multicast_lock_acquire = 0;
```

Comment:
Following the above comments, these would also need to move into a matching wrapper file. It seems the `NetSocketAndroid` would be the prime location. Is there a reason why these are not defined there?